### DISCUSSION TITLE
大家导入知识库的文本是什么样子的？

### DISCUSSION THREAD

**Discussion by sealot** [chatchat-space/Langchain-Chatchat#1480] [Q&A]
听说导入文本质量很很重要，最好是Q&A形式的，有没有同学可以分享一下，好的知识库内容应该什么样子？比如：
问：植物光合作用吸收的什么能源？
答：光能。

类似这样的，一问一答，很多问题放在一个文件里面吗？

**Comment by Alien-smile** [comment]
同问。
如果不是QA形式，感觉回答的很难准确

**Comment by zRzRzRzRzRzRzR** [comment]
自定义分词器，可以有效的提升准确度，wiki 的最佳实践中也有可以配置较好的模型解决这个问题
